Consistently mark required fields of mapping schemas (#159)
home-assistant-libs/voluptuous-serialize: README.md

    "type": "string",
    "default": "not specified",
    "required": false,

>     "optional": true,  // This will be deprecated soon. Please use "required" key instead.

Reviewer: ```suggestion
    "optional": true,  // This is deprecated. Please use "required" key instead.
```
